debian/control

```diff
@@ -54,11 +54,14 @@ Architecture: any
 Depends: adduser,
          apparmor (>= 2.10.95-0ubuntu2.2),
          ca-certificates,
+         cgroup-lite,
          gnupg1 | gnupg,
          snap-confine (= ${binary:Version}),
          squashfs-tools,
-         systemd,
+         sudo (>=1.8.9p5-1ubuntu1.3),
+         systemd(>=204-5ubuntu20.20),
          ubuntu-core-launcher (= ${binary:Version}),
+         util-linux (>=2.20.1-5.1ubuntu20.9),
```

Comment:
This package is `Essential: yes` so we don't need to depend on it. We only need it to ensure the version number on trusty is right. Would be nice to add a small comment in here `# needed to pull in the right version on trusty` or something.